homework4.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
+console.log('it works');
+
+
+$('.readmore, .readless, .learnmore').click(function(event){
```

Comment:
For readability's sake, indent everything 1 tab that's inside the `$(document).ready` function.

```js
$(document).ready(function() {
  // indented 1 tab
  console.log('it works');
  $('.readmore, .readless, .learnmore').click(function(event){
    // indented 2 tabs
  });
});

```